checklists/alz_checklist.en.json

```diff
@@ -2422,6 +2422,7 @@
             "guid": "cdb3751a-b2ab-413a-ba6e-55d7d8a2adb1",
             "id": "G02.07",
             "severity": "Medium",
+            "graph" : "Resources | where type =~ 'microsoft.keyvault/vaults'  | extend properties = parse_json(properties)  | project id, name, location, firewallEnabled = properties.networkAcls.defaultAction, privateEndpointConnections = properties.privateEndpointConnections  | extend compliant = iff(firewallEnabled == 'Deny' or array_length(privateEndpointConnections) > 0, 1, 0)  | where compliant == 1  | project id, compliant",
```

Comment:
@jodyford-msft  - So this query is only going to show the ones that are compliant, but not the non-compliant ones, from my understanding.  I think we want to take out the "where compliant" to show all.

I was liking the output of:

`Resources | where type =~ 'microsoft.keyvault/vaults'  | extend properties = parse_json(properties)  | project id, name, location, firewallEnabled = properties.networkAcls.defaultAction, privateEndpointConnections = properties.privateEndpointConnections  | extend compliant = iff(firewallEnabled == 'Deny' or array_length(privateEndpointConnections) > 0, 1, 0)  | project id, compliant, name, location`